pkgs/applications/terminal-emulators/wezterm/default.nix

```diff
@@ -0,0 +1,87 @@
+{ rustPlatform
+, lib
+, fetchFromGitHub
+
+, pkgconfig
+, fontconfig
+, python3
+, openssl
+, perl
+
+, dbus
+, libX11
+, xcbutil
+, libxcb
+, xcbutilkeysyms
+, xcbutilwm # contains xcb-ewmh among others
+, libxkbcommon
+, libglvnd # libEGL.so.1
+, egl-wayland
+, wayland
+, libGLU
+, libGL
+, freetype
+, zlib
+}:
+let
+  runtimeDeps = [
+    libX11
+    xcbutil
+    libxcb
+    xcbutilkeysyms
+    xcbutilwm
+    libxkbcommon
+    dbus
+    libglvnd
+    zlib
+    egl-wayland
+    wayland
+    libGLU
+    libGL
+    fontconfig
+    freetype
+  ];
+
+  pname = "wezterm";
+  version = "3bd8d8c84591f4d015ff9a47ddb478e55c231fda";
+in
+rustPlatform.buildRustPackage {
+  inherit pname;
+  inherit version;
+
+  src = fetchFromGitHub {
+    owner = "wez";
+    repo = pname;
+    rev = "${version}";
+    fetchSubmodules = true;
```

Comment:
I would prefer it if we would use the system libraries instead of the vendored ones.

Reply:
Assuming that your goal is to remove this line entirely I think this isn't easily possible. The submodule paths are referenced unconditionally, e.g. here: 

https://github.com/wez/wezterm/blob/5fbb6a44002c170dcd20bbc0b0fd38f6fbf6b206/wezterm-font/Cargo.toml#L15